conf/application.conf

```diff
@@ -225,6 +225,12 @@ microservice {
       port = 9508
       enabled = true
     }
+
+    transit-movements-upscan-initiate {
+      protocol = "http"
+      host = "localhost"
+      port = 9570
+    }
```

Comment:
We'll need to add this to the `app-config-base` repository as well with the correct remote values.